### PR TITLE
ColorBox: add a Getting Started tutorial

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/00 Getting Started with ColorBox.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#include tutorials-intro-installationnote
+
+The ColorBox component allows users to enter a color or pick it out from the drop-down editor.
```
Comment (dmitry-eliseev-devexpress) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/00 Getting Started with ColorBox.md]: Why "pick out"? A color picker (color box or whatever) allows one to pick a color from it. Nothing else.

From **a** drop-down editor. We're introducing it, we haven't seen it yet.

```suggestion
The ColorBox component allows users to enter a color or pick it from a drop-down editor.
```

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/00 Getting Started with ColorBox.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+This tutorial shows how to add a ColorBox component to the page and configure the component's core settings. As a result, you will create the following UI component:
```
Comment (dmitry-eliseev-devexpress) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/00 Getting Started with ColorBox.md]: Why to **the** page? The current page? One can complete this tutorial to be able to add the component to **any** page they may want.

```suggestion
This tutorial shows how to add a ColorBox component to a page and configure the component's core settings. As a result, you will create the following UI component:
```

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/10 Configure Values.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+The ColorBox accepts colors in hexadecimal (_"#FF0000"_), RGB (_"rgb(255, 0, 0)"_) and RGBA (_"rgba(255, 0, 0, 1)"_) formats, as well as <a href="http://www.w3schools.com/cssref/css_colors.asp" target="_blank">color names</a>. When users select a color from the drop-down editor, the textual representation of the selected color is hexadecimal.
```
Comment (dmitry-eliseev-devexpress) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/10 Configure Values.md]: Don't forget about Oxford commas.

Representation. I hoped I'll never hear this from you: http://tw.devexpress.devx/language-guidelines/misused-words/represent/

Please do not forget that every "represent" in your text says of your negligence, laziness, or both.

```suggestion
The ColorBox accepts colors in hexadecimal (_"#FF0000"_), RGB (_"rgb(255, 0, 0)"_), and RGBA (_"rgba(255, 0, 0, 1)"_) formats, as well as <a href="http://www.w3schools.com/cssref/css_colors.asp" target="_blank">color names</a>. When users select a color from the drop-down editor, the ColorBox displays the selected color as a string in hexadecimal format.
```

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/10 Configure Values.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+To allow users to control the transparency of the selected color, enable the [editAlphaChannel](/api-reference/10%20UI%20Components/dxColorBox/1%20Configuration/editAlphaChannel.md '/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#editAlphaChannel') property. This setting adds a slider that regulates transparency to the drop-down editor, and changes the textual representation of the selected color from hexadecimal to RGBA.
```
Comment (dmitry-eliseev-devexpress) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/10 Configure Values.md]: > This setting adds a slider that regulates transparency to the drop-down editor, and changes the textual representation of the selected color from hexadecimal to RGBA.

This sentence is bad, not only due to "representation". You can do better. Please rewrite.

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/10 Configure Values.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+To specify the initial picked color, use the [value](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#value) property.
```
Comment (dmitry-eliseev-devexpress) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/10 Configure Values.md]: Initial picked color or initially picked color?

```suggestion
To specify the initially picked color, use the [value](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#value) property.
```

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
+Specify the following properties to add custom buttons into the input field, or configure the drop-down editor's buttons:
```
Comment (dmitry-eliseev-devexpress) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md]:
```suggestion
Specify the following properties to add custom buttons to the input field, or configure the drop-down editor's buttons:
```

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+- [buttons](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/buttons/)    
+Allows you to add custom buttons to the input text field.
+
+- [showClearButton](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#showClearButton)    
+Specifies whether to display the Clear button in the ColorBox.
+
+- [applyButtonText](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#applyButtonText)    
+Specifies the text for the button that applies changes and closes the drop-down editor.
```
Comment (dmitry-eliseev-devexpress) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md]: the text for the button == caption?

Be simpler.

```suggestion
Specifies the caption of the button that applies changes and closes the drop-down editor.
```

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+
+- [cancelButtonText](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#cancelButtonText)    
+Specifies the text for the button that cancels changes and closes the drop-down editor.
```
Comment (dmitry-eliseev-devexpress) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md]:
```suggestion
Specifies the caption of the button that cancels changes and closes the drop-down editor.
```

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+
+You can hide the editor's drop-down button to allow users to only type in a color code. To do this, disable the [showDropDownButton](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#showDropDownButton) property.
```
Comment (dmitry-eliseev-devexpress) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md]:
```suggestion
You can hide the editor's drop-down button to limit user input options down to color code entry. To do this, disable the [showDropDownButton](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#showDropDownButton) property.
```

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+
+To add a label to the ColorBox, specify the [label](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#label) property. To change the label appearance, set the [labelMode](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#labelMode) property to one of the following values:
+
+- *"static"* (default)    
+The component displays the label above the input field.
+
+- *"floating"*    
+The component uses the label as a placeholder, but when the editor receives focus, the label moves to the position above the input field.
```
Comment (dmitry-eliseev-devexpress) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md]: The simpler your punctuation, the better.

```suggestion
The component uses the label as a placeholder but the label moves to the position above the input field when the editor receives focus.
```

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+
+- *"hidden"*    
+The label is hidden.
+
+The following code displays a Clear button, changes the Apply button text and specifies a label for the ColorBox.
```
Comment (dmitry-eliseev-devexpress) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md]:
```suggestion
The following code displays a Clear button, changes the Apply button's caption and specifies a label for the ColorBox.
```

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/20 Handle Value Change.md [hunk 12]
```diff
@@ -0,0 +1,128 @@
+Users can change the value after they pick a shade, or after they click the **OK** button. To specify the mode, use the [applyValueMode](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#applyValueMode) property. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#onValueChanged) function to handle the value change when users select a color in the drop-down editor or type it in.
```
Comment (dmitry-eliseev-devexpress) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/20 Handle Value Change.md]:
```suggestion
Users can change the value after they pick a shade or after they click the **OK** button. To switch between value change modes, use the [applyValueMode](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#applyValueMode) property. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#onValueChanged) function to handle value changes when users select a color in the drop-down editor or type a color code in.
```

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/20 Handle Value Change.md [hunk 13]
```diff
@@ -0,0 +1,128 @@
...
+
+The following code displays a notification every time users change the ColorBox value. The background color of the notification is the color they picked.
```
Comment (dmitry-eliseev-devexpress) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/20 Handle Value Change.md]:
```suggestion
The following code displays a notification every time users change the ColorBox value. The notification's background assumes the picked color. 
```

**Review by dmitry-eliseev-devexpress — COMMENTED** [review]
1. Ask yourself correct questions to choose articles correctly. Of what do we speak right now and why?
2. No representation in our documentation. No exceptions.
3. Please be simpler and do everything you can to avoid sentence overcomplication.

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/10 Configure Values.md (hunk 4 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/10 Configure Values.md]: I rewrote the sentence

Comment (dmitry-eliseev-devexpress) [reply concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/10 Configure Values.md]: Thank you. Everything looks fine now.

**Review by dmitry-eliseev-devexpress — APPROVED** [review]

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+- [showClearButton](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#showClearButton)    
+Specifies whether to display the Clear button in the ColorBox.
+
+- [applyButtonText](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#applyButtonText)    
+Specifies the caption of the button that applies changes and closes the drop-down editor.
+
+- [cancelButtonText](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#cancelButtonText)    
+Specifies the caption of the button that cancels changes and closes the drop-down editor.
+
+You can hide the editor's drop-down button to limit user input options down to color code entry. To do this, disable the [showDropDownButton](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#showDropDownButton) property.
```
Comment (albertov05) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md]:
```suggestion
You can hide the editor's drop-down button to limit user input options to color code entry. To do this, disable the [showDropDownButton](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#showDropDownButton) property.
```

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md [hunk 15]
```diff
@@ -0,0 +1,86 @@
...
+
+To add a label to the ColorBox, specify the [label](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#label) property. To change the label appearance, set the [labelMode](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#labelMode) property to one of the following values:
+
+- *"static"* (default)    
+The component displays the label above the input field.
+
+- *"floating"*    
+The component uses the label as a placeholder but the label moves to the position above the input field when the editor receives focus.
```
Comment (albertov05) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md]:
```suggestion
The component uses the label as a placeholder, but the label moves to the position above the input field when the editor receives focus.
```

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md [hunk 16]
```diff
@@ -0,0 +1,86 @@
...
+
+- *"hidden"*    
+The label is hidden.
+
+The following code displays a Clear button, changes the Apply button's caption and specifies a label for the ColorBox.
```
Comment (albertov05) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/15 Customize Buttons and Labels.md]:
```suggestion
The following code displays a Clear button, changes the Apply button's caption, and specifies a label for the ColorBox.
```

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/20 Handle Value Change.md [hunk 17]
```diff
@@ -0,0 +1,128 @@
+Users can change the value after they pick a shade or after they click the **OK** button. To switch between value change modes, use the [applyValueMode](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#applyValueMode) property. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#onValueChanged) function to handle value changes when users select a color in the drop-down editor or type a color code in.
```
Comment (albertov05) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/20 Handle Value Change.md]:
```suggestion
Users can change the value after they pick a shade or after they click the **OK** button. To switch between value change modes, use the [applyValueMode](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#applyValueMode) property. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#onValueChanged) function to handle value changes when users select a color in the drop-down editor or type a color code.
```

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/10 Configure Values.md [hunk 18]
```diff
@@ -0,0 +1,60 @@
+The ColorBox accepts colors in hexadecimal (_"#FF0000"_), RGB (_"rgb(255, 0, 0)"_), and RGBA (_"rgba(255, 0, 0, 1)"_) formats, as well as <a href="http://www.w3schools.com/cssref/css_colors.asp" target="_blank">color names</a>. When users select a color from the drop-down editor, the ColorBox displays the selected color as a string in hexadecimal format.
+
+To allow users to control the transparency of the selected color, enable the [editAlphaChannel](/api-reference/10%20UI%20Components/dxColorBox/1%20Configuration/editAlphaChannel.md '/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#editAlphaChannel') property. If you enable this property, the component displays the selected color in RGBA format.
+
+To specify the initially picked color, use the [value](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#value) property.
```
Comment (albertov05) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/10 Configure Values.md]:
```suggestion
To specify the initially selected color, use the [value](/Documentation/ApiReference/UI_Components/dxColorBox/Configuration/#value) property.
```

File: concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/20 Handle Value Change.md [hunk 19]
```diff
@@ -0,0 +1,128 @@
...
+
+The following code displays a notification every time users change the ColorBox value. The notification's background assumes the picked color.
```
Comment (albertov05) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/20 Handle Value Change.md]: `The notification's background assumes the picked color.`

Can we say the following instead?

`The selected color is applied to the notification's background.`

Comment (vladaskorohodova) [inline concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/20 Handle Value Change.md]:
```suggestion
The following code displays a notification every time users change the ColorBox value. The selected color is applied to the notification's background.
```

Comment (vladaskorohodova) [reply concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/20 Handle Value Change.md]: Yes, I changed it

Comment (albertov05) [reply concepts/05 UI Components/ColorBox/00 Getting Started with ColorBox/20 Handle Value Change.md]: Thank you!

**Pull request merged**